include/utils/system_reference_blas.hpp

```diff
@@ -257,6 +271,14 @@ void gemm(const char *transA, const char *transB, int m, int n, int k,
        lda, b, ldb, beta, c, ldc);
 }
 
+template <typename scalar_t>
+void trsm(char side, char triangle, char transpose, char diagonal, int m, int n,
```

Comment:
I don't think this is part of the intended MR, I suppose this just needs a rebase on top of latest branch changes?

Reply:
I will update this with Ben's commit. It needs some changes before merging